plasmapy/physics/transport.py

```diff
@@ -219,25 +219,34 @@ def Coulomb_logarithm(T,
     return ln_Lambda
 
 
-def _boilerPlate(T, particles, V):
-    """
-    Some boiler plate code for checking if inputs to functions in
-    transport.py are good. Also obtains reduced in mass in a
-    2 particle collision system along with thermal velocity.
-    """
+"""
+  Some code for checking if inputs to functions in
+  transport.py are good and for obtaining reduced mass in a
+  2 particle collision system along with thermal velocity.
+"""
+
+
+def checkTemperature(T):
     # checking temperature is in correct units
     T = T.to(u.K, equivalencies=u.temperature_energy())
-    # extracting particle information
+    return T
+
+
+def checkParticleSystem(particles):
+    # checking particles input
     if not isinstance(particles, (list, tuple)) or len(particles) != 2:
         raise ValueError("Particles input must be a "
                          "list or tuple containing representations of two  "
                          f"charged particles. Got {particles} instead.")
+    return particles
 
+
+def getParticleSystemInfo(particles):
```

Comment:
I think the two functions above could be merged. They're not doing anything special enough on their own to warrant being separate.